### ISSUE TITLE
Using KR language of tui editor in next JS is not working.

### ISSUE THREAD

**Issue by Yangseungchan** [nhn/tui.editor#1677]
## Describe the bug
I'm creating NextJS app and I finished importing the tui editor in the page and it is rendered successfully. After that, I am trying to apply KR languages into tooltips according to this [guideline](https://github.com/nhn/tui.editor/blob/master/docs/en/i18n.md). However, importing '@toast-ui/editor/dist/i18n/ko-kr'  is not working and also setting **language** props as 'ko' is not working as well. Is there any solution to apply kr language?

## Screenshots
![importing ko-kr package not working](https://user-images.githubusercontent.com/38908080/125410744-2eb5e800-e3f8-11eb-86e6-19c600f3c849.png)

## Desktop (please complete the following information):
 - OS: Mac OS
 - Browser: Chrome
 - Version: 22

**Comment by js87zz** [comment]
@Yangseungchan 
I think it occurs due to our `exports` field in `package.json`. Until we fix the problem, it's better to set i18n directly as below.
```js
Editor.setLanguage(['ko', 'ko-KR'], {
  Markdown: '마크다운',
  WYSIWYG: '위지윅',
  Write: '편집하기',
  Preview: '미리보기',
  Headings: '제목크기',
  Paragraph: '본문',
  Bold: '굵게',
  Italic: '기울임꼴',
  Strike: '취소선',
  Code: '인라인 코드',
  Line: '문단나눔',
  Blockquote: '인용구',
  'Unordered list': '글머리 기호',
  'Ordered list': '번호 매기기',
  Task: '체크박스',
  Indent: '들여쓰기',
  Outdent: '내어쓰기',
  'Insert link': '링크 삽입',
  'Insert CodeBlock': '코드블럭 삽입',
  'Insert table': '표 삽입',
  'Insert image': '이미지 삽입',
  Heading: '제목',
  'Image URL': '이미지 주소',
  'Select image file': '이미지 파일을 선택하세요.',
  'Choose a file': '파일 선택',
  'No file': '선택된 파일 없음',
  Description: '설명',
  OK: '확인',
  More: '더 보기',
  Cancel: '취소',
  File: '파일',
  URL: '주소',
  'Link text': '링크 텍스트',
  'Add row to up': '위에 행 추가',
  'Add row to down': '아래에 행 추가',
  'Add column to left': '왼쪽에 열 추가',
  'Add column to right': '오른쪽에 열 추가',
  'Remove row': '행 삭제',
  'Remove column': '열 삭제',
  'Align column to left': '열 왼쪽 정렬',
  'Align column to center': '열 가운데 정렬',
  'Align column to right': '열 오른쪽 정렬',
  'Remove table': '표 삭제',
  'Would you like to paste as table?': '표형태로 붙여 넣겠습니까?',
  'Text color': '글자 색상',
  'Auto scroll enabled': '자동 스크롤 켜짐',
  'Auto scroll disabled': '자동 스크롤 꺼짐',
  'Choose language': '언어 선택',
});

const editor = new Editor({
  // ...
  language: 'ko'
});
```

I'll let you know when it's fixed!

**Comment by Yangseungchan** [comment]
Thanks for suggestion. However, as I'm using Editor as a react component from **@toast-ui/react-editor**, there seems to be no method you mentioned. Do you have any other suggestion until it is fixed ?

**Comment by js87zz** [comment]
@Yangseungchan 
Regardless of the react, the static method can be used as below.
```js
import ToastUIEditor from @toast-ui/editor;
import Editor from @toast-ui/react-editor;

ToastUIEditor. setLanguage(...)

// ...
```

**Comment by Yangseungchan** [comment]
I applied what you said but it seems that korean is not applied to my editor.  Can you please look at my [code](https://codesandbox.io/s/dazzling-microservice-lyp36) and give me suggestion? Thanks for caring my questions :)

**Comment by js87zz** [comment]
@Yangseungchan 
I found the typo in `Editor`.(`langauage` => `language`)

**Comment by partoneplay** [comment]
waiting ...

**Comment by bradyisom** [comment]
I ran into this same issue. @js87zz, good work on making the fix. Any idea on when this fix will be pushed to NPM? My I18N is effectively broken until I can get this fix.

**Comment by js87zz** [comment]
@bradyisom @partoneplay @Yangseungchan 
The bug has been fixed in `v3.0.2`. Thanks!
